Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -586,10 +586,10 @@ function Add-GroupToCloudAdmins {
 
 <#
     .Synopsis
-     Remove a previously added external identity group from the cloud admin group
+     Remove a previously added group from an external identity from the vsphere CloudAdmins group
```

Comment:
I suggest  just 'the CloudAdmins group' here for consistency with other descriptions.